src/root/views/Country/ThreeW/ProjectDetailModal/index.tsx

```diff
@@ -0,0 +1,251 @@
+import React from 'react';
+import {
+  _cs,
+  isNotDefined,
+} from '@togglecorp/fujs';
+
+import BlockLoading from '#components/block-loading';
+import BasicModal from '#components/BasicModal';
+import Header from '#components/Header';
+import Button, { Props as ButtonProps } from '#components/Button';
+import TextOutput, { Props as TextOutputProps } from '#components/TextOutput';
+import { useRequest } from '#utils/restRequest';
+import { Project } from '#types';
+
+import styles from './styles.module.scss';
+
+function FieldOutput(props: TextOutputProps) {
+  const {
+    className,
+    ...otherProps
+  } = props;
+
+  return (
+    <TextOutput
+      className={_cs(styles.fieldOutput, className)}
+      labelContainerClassName={styles.label}
+      valueContainerClassName={styles.value}
+      {...otherProps}
+    />
+  );
+}
+
+interface RowProps {
+  className?: string;
+  children?: React.ReactNode;
+}
+function Row(props: RowProps) {
+  const {
+    className,
+    children,
+  } = props;
+
+  return (
+    <div
+      className={_cs(
+        styles.row,
+        className,
+      )}
+    >
+      { children }
+    </div>
+  );
+}
+
+interface Props {
+  className?: string;
+  projectId: number;
+  headerActions?: React.ReactNode;
+  onCloseButtonClick?: ButtonProps<void>['onClick'];
+}
+
+function ProjectDetail(props: Props) {
+  const {
+    className,
+    projectId,
+    headerActions,
+    onCloseButtonClick,
+  } = props;
+
+  const {
+    pending: projectPending,
+    response: projectResponse,
+  } = useRequest<Project>({
+    skip: isNotDefined(projectId),
+    url: `api/v2/project/${projectId}`,
+  });
+
+  if (isNotDefined(projectId)) {
+    return null;
+  }
+
+  return (
+    <BasicModal
+      className={_cs(styles.projectDetail, className)}
+      hideCloseButton
+    >
+      <Header
+        className={styles.header}
+        headingSize="extraLarge"
+        heading={projectResponse?.name}
+        actions={(
+          <>
+            { headerActions }
+            <Button
+              variant="secondary"
+              onClick={onCloseButtonClick}
+            >
+              Close
+            </Button>
+          </>
+        )}
+        descriptionClassName={styles.headerDescription}
+        description={(
+          <TextOutput
+            label="Last Modified on"
```

Comment:
Add fixme to use string or use strings

Reply:
We'll be adding strings for FieldReport and 3W in bulk